xrpl/asyncio/transaction/multisign.py

```diff
@@ -0,0 +1,73 @@
+"""Multisign transaction methods with XRPL transactions."""
+from typing import List
+
+from xrpl.core.binarycodec import decode, encode, encode_for_multisigning
+from xrpl.core.keypairs.main import sign
+from xrpl.models.transactions.transaction import Signer, Transaction
+from xrpl.wallet.main import Wallet
+
+
+async def sign_for_multisign(
```

Comment:
I actually preferred the way that we did it in xrpl.js - when doing a multisign, the normal `sign` method has an extra param.